etc/tfw_sched_http.conf

```diff
@@ -1,157 +0,0 @@
-#
```

Comment:
Did we considered to merge the config with main config? Probably, it's better to keep it separate to separate reverse proxy logic.